app/assets/stylesheets/application/homepage/_event_thumbnail.scss

```diff
@@ -0,0 +1,55 @@
+.event-thumbnail {
+  background-color: $secondary-background-color;
+  vertical-align: top;
+  text-align: left;
+  position: relative;
+  display: inline-block;
+  margin: 20px 7px;
+  width: 292px;
+
+  &:hover {
+    background-color: #ece9e7;
+  }
+
+  a {
+    text-decoration: none;
+    color: black;
+  }
+
+  img {
+    object-fit: cover;
+    height: 194px;
+    width: 292px;
+  }
+
+  h2 {
+    word-spacing: 2px;
+    font-family: $secondary-font;
+    line-height: 1.2em;
+    font-size: 1.2em;
+    padding: 7px 14px 0;
+  }
+
+  p {
+    padding: 7px 14px;
+  }
+
+  .venue {
+    padding-bottom: 14px;
+    font-style: italic;
+  }
+
+  .details {
+    padding: 8px 14px 4px;
+
+    .price {
+      background-color: $primary-color;
+      font-family: $secondary-font;
```

Comment:
Properties should be ordered background-color, color, font-family, left, padding, position, top